shared/android/app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,11 @@
     <uses-permission android:name="android.permission.READ_CONTACTS" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
+    <meta-data  android:name="com.dieam.reactnativepushnotification.notification_channel_name"
```

Comment:
@MarcoPolo maybe not needed after your changes?

Reply:
@MarcoPolo bump

Reply:
Oh yeah, kill this